core/src/main/java/com/segment/analytics/kotlin/core/Storage.kt

```diff
@@ -98,11 +101,16 @@ fun parseFilePaths(filePathStr: String?): List<String> {
  *  provider via this interface
  */
 interface StorageProvider {
+    @Deprecated("Deprecated in favor of create which takes vararg params",
+        ReplaceWith("createStorage(analytics, store, writeKey, ioDispatcher, application)")
```

Comment:
We say to replace it with createStorage with all the parameters, but are there any implementations of createStorage that use anything other than the Analytics param?

Reply:
That is, should it just be `ReplaceWith("createStorage(analytics)")`